README.md

```diff
@@ -1,10 +1,31 @@
 # Arachnivania
 
-This project is a Godot-powered game using the native GDscript language. Art is created using Aseprite.
+This project is a Godot-powered game using the native GDscript language. Art is created using Aseprite, TileSetter, and GIMP.
+
+[1. Description](#desription)
+[2. Controls](#controls)
+[3. Build Info](#build-info)
+[4. Authors](#license)
+[5. License](#license)
+[6. Acknowledgements](#acknowledgements)
 
 ## Description
 
-Arachnivania is a 2D Metroidvania. In it you play as a spider trying to protect its burrow from invading Wasps. In the process you will discover new ways to fight and avoid enemies while trying to safe guard your burrow.
+Arachnivania is a 2D Metroidvania. You play as Widower, a warrior spider trying to protect his burrow from the invading Wasps. In the process you will discover new ways to fight and avoid enemies while protecting your family and burrow from harm.
 
 ## Controls
 
+* WASD - used for basic movement, A & D move left and right, respectively, while W & S move the camera slightly up or down.
+* SPACE - used to jump over obstacles and enemies
```

Comment:
The prototype currently uses W as jump, just fyi

Reply:
oh okay thats fine, didn't know.

Reply:
Oh yeah, you can link to whatever files you want with Markdown. What those links do at the top is scroll to the section of the page that is linked. So if you click on Acknowledgements at the top, its scrolls to the Acknowledgements part of the README

Reply:
No its fine, it doesnt really matter so long as it represents the final game